### ISSUE TITLE
Chakra ui v3

### ISSUE THREAD

**Issue by MastalerzKamil** [horizon-ui/horizon-ui-chakra-pro-ts#4]
Are you support latest chakra ui v3. Seems that there are breaking changes and we purchased lifetime licence so I could expect update on your site

**Comment by ArintonAkos** [comment]
I agree, we also want to migrate to Chakra UI V3. Any news regarding this? @VladMihalache @simmmpleweb
